draft-song-dns-wireformat-http-01.txt

```diff
@@ -295,7 +298,7 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
    functionality.
 
    For a stub resolver that connect directly via HTTP to the HTTP server
-   proxy then this flag is not necessary, as the entire response can
+   proxy then this flag should be set to TCP, as the entire response can
```

Comment:
This edit/comment reminds me that we missed one case that mixed scenario that only one side ,Client or Server implement as a proxy.   In a protocol design, we should make clear the client behavior and server behavior separately.